frontend/public/components/monitoring/alert-manager-config.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { safeLoad } from 'js-yaml';
+import {Base64} from 'js-base64';
```

Comment:
Nit: Inconsistent spacing for `import { ... }`. I think we generally put space chars inside the `{}`. (Maybe we can add an eslint rule.)